airbyte-integrations/connectors/source-hubspot/source_hubspot/spec.yaml

```diff
@@ -86,6 +86,11 @@ connectionSpecification:
                 if you need help finding this token.
               type: string
               airbyte_secret: true
+    enable_experimental_streams:
```

Comment:
why do we need this property in configuration if user can select desired streams during the connection setup?

Reply:
This is because of agreement. We agreed with @katmarkham that this would be a good way to make experimental streams turned on. Those streams are based on beta API and may be a cause problems. We had discussion with some proposals and desisions, please take a look: https://airbyte-globallogic.slack.com/archives/C02TL38U5L7/p1700667063084059